stdlib/public/SwiftShims/RuntimeStubs.h

```diff
@@ -31,6 +31,11 @@ SWIFT_RUNTIME_STDLIB_INTERFACE
 __swift_ssize_t
 swift_stdlib_readLine_stdin(char * _Nullable * _Nonnull LinePtr);
 
+SWIFT_RUNTIME_STDLIB_INTERFACE
+char * _Nullable * _Nonnull
+_swift_get_args(int * _Nonnull outArgLen);
```

Comment:
It should be prefixed with `_swift_stdlib_*`, and `get_args` is not very descriptive.  Something like `_swift_stdlib_getProcessArguments` would be better.